src/test/java/com/commercetools/pingpong/model/PlayerTest.java

```diff
@@ -1,29 +1,79 @@
 package com.commercetools.pingpong.model;
 
+import org.assertj.core.error.ShouldBe;
+import org.assertj.core.error.ShouldBeEqual;
+import org.junit.Before;
 import org.junit.Test;
 
+import javax.xml.stream.util.EventReaderDelegate;
```

Comment:
What do we need this for?

Reply:
Good question 😅 
I removed it 👍 
db130fba9869d4f4af97c9d4ff58d7a474ac9acf